### PR TITLE
MCP-2501 PDF Generator CSS File Setup

### PR DESCRIPTION
**Pull request by anthonyvelazquez**
## What was the problem?

All the styling was inline. It makes the template harder to read and also troubleshoot if it requires updates in the future

Associated tickets or Slack threads:

- [MCP-2501](https://amida.atlassian.net/browse/MCP-2501)

## How does this fix it?[^1]

CSS styling was moved to a separate `base.css` file. Not everything was moved, column widths were left as inline styles for now

## How to test this PR
- Generate a PDF based off the `develop` branch and then generate the same PDF off of this branch. Both should match

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Comment by alan-amida** [comment]
Oh, didn't realize this was still in Draft, sorry. Might have messed up your timeline.

**Comment by anthonyvelazquez** [comment]
> Oh, didn't realize this was still in Draft, sorry. Might have messed up your timeline.

All good, was waiting on the Font PR to be merged since it had work needed for this. Will get this ready tomrrow

**Comment by alan-amida** [comment]
end2end failures:
- https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/4430315922/jobs/7771894112#step:8:1063
> VroV1Tests > positive7101Case01() FAILED
    org.springframework.web.client.HttpServerErrorException$InternalServerError: 500 : "{"claimSubmissionId":"7001","status":"ERROR","reason":"[Errno 2] No such file or directory: '/home/docker/lib/templates/hypertension-v1/base.css'"}"
- https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/4430315922/jobs/7771894112#step:8:1099
> VroV1Tests > positive6602Case01() FAILED
    org.springframework.web.client.HttpServerErrorException$InternalServerError: 500 : "{"claimSubmissionId":"7002","status":"ERROR","reason":"[Errno 2] No such file or directory: '/home/docker/lib/templates/asthma-v1/base.css'"}"

**Comment by jbregstone** [comment]
Everything looks good except the first page looks different. Name table has different dimension and the links are black rather than blue. also the Table of Contents goes onto the second page. 
[VAMC_Hypertension_Rapid_Decision_Evidence--20230316.pdf](https://github.com/department-of-veterans-affairs/abd-vro/files/10993559/VAMC_Hypertension_Rapid_Decision_Evidence--20230316.pdf)

File: service-python/pdfgenerator/src/lib/templates/shared/base.css
Comment (alan-amida) [inline service-python/pdfgenerator/src/lib/templates/shared/base.css:1]: Forgot to clean this file up, or is it meant to stay and be empty?

**Pull request closed without merging**